### ISSUE TITLE
Add CodeBuddies as a beginner-friendly open-sourced project option

### ISSUE THREAD

**Issue by lpatmo** [yourfirstpr/yourfirstpr.github.io#50]
CodeBuddies is a community of independent code learners who enjoy sharing knowledge and helping each other learn faster. We come from all over the world! Everyone is welcome, independent of previous knowledge.

The new app that we built, which we're currently beta-ing (and fixing up issues) before we officially roll out, is 100% community-built by volunteers from around the world. The app helps the community organize google hangouts to study and learn together.

We have a list of beginner-friendly issues labeled under the `good-for-beginners` tag here: https://github.com/codebuddiesdotorg/cb-v2-scratch/issues?q=is%3Aissue+is%3Aopen+label%3Agood-for-beginners

We also have a comprehensive contributing.md doc that walks you through how to clone the app and get it set up: https://github.com/codebuddiesdotorg/cb-v2-scratch/blob/master/contributing.md

The app is built on meteorJS, and the issues delve into HTML/CSS, javascript, and beginner MeteorJS.

We are also pretty active on our codebuddiesmeet.slack.com slack, where you can ask questions and get responses on a number of programming topics. The #codebuddies-meta channel is where we publicly discuss and write updates about the project.

**Comment by lpatmo** [comment]
We just updated all our beginner-friendly issues with the `need-a-hero` label: https://github.com/codebuddiesdotorg/cb-v2-scratch/issues?q=is%3Aissue+is%3Aopen+label%3A%22needs+a+hero%22 

:)

**Comment by sunilsachdeva** [comment]
Slack address has changed from codebuddiesmeet.slack.com to codebuddies.slack.com

**Comment by lpatmo** [comment]
@sunilsachdeva Yes, that's right! (Technically codebuddiesmeet still redirects to codebuddies, but that could change at any time.) Slack self-signup is still on the home page at codebuddies.org.
